data-prepper-plugins/kinesis-source/src/main/java/org/opensearch/dataprepper/plugins/kinesis/source/configuration/ConsumerStrategy.java

```diff
@@ -0,0 +1,23 @@
+package org.opensearch.dataprepper.plugins.kinesis.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+// Reference: https://docs.aws.amazon.com/streams/latest/dev/enhanced-consumers.html
+
+public enum ConsumerStrategy {
+
+    POLLING("Polling"),
```

Comment:
```suggestion
    POLLING("polling"),
```

The Data Prepper convention is to use strings with all lowercase.

Reply:
Thanks @dlvenable. I have fixed this.